_posts/2021-02-24-#21224.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: pr
+date: 2021-02-24
+title: "Halt processing of unrequested transactions"
+pr: 21224
+authors: [ariard]
+components: ["p2p"]
+host: ellemouton
+status: upcoming
+commit:
+---
+
+## Notes
+
+- Transaction relay is a three step process: `inv` -> `getdata` -> `tx`:
+
+  1. the relaying node sends an `inv` message to the receiving node to announce
+     a new transaction.
+  2. the receiving node sends a `getdata` message to the relaying node to
```

Comment:
Perhaps add something like "if the receiving node wants to retrieve the transaction..." or similar. The receiving node won't request transactions that it's seen before, and will only request from one peer if it receives invs for the same transaction from multiple peers.